coffee_cmd/src/coffee/mod.rs

```diff
@@ -176,7 +176,25 @@ impl PluginManager for CoffeeManager {
         Err(err)
     }
 
-    async fn list(&mut self) -> Result<(), CoffeeError> {
+    async fn list(&mut self, remotes: bool) -> Result<(), CoffeeError> {
+        let mut plugin_names = Vec::new();
+        if remotes {
+            for repo in &self.repos {
+                plugin_names.append(
+                    &mut repo
+                        .list()
+                        .await
+                        .unwrap()
+                        .iter()
+                        .map(|p| p.name())
+                        .collect(),
```

Comment:
wow this is ugly, sorry

Reply:
I tried spliting the command into 2 lines.